### ISSUE TITLE
I am getting an error "Cannot find module .."

### ISSUE THREAD

**Issue by gj1118** [sohelamin/electron-vue-boilerplate#5]
Hi Sohail, 
I am getting an error as follows 

```
Uncaught Error: Cannot find module "./src/app.js"
    at webpackMissingModule (build.js:11529)
    at Object.<anonymous> (build.js:11529)
    at __webpack_require__ (bootstrap 4c93556…:659)
    at bootstrap 4c93556…:708
    at bootstrap 4c93556…:708
```

I get this error whenever I do , npm run dev-server and in another tab I do npn run dev .

I replaced your app.js with my app.js which is about the same as yours except mine has a few more Vue routes . I have also made sure that the file app.js exists in the src folder. 

**Update**
I am even getting the same error with your version. Please see the snapshot below. 
![image](https://cloud.githubusercontent.com/assets/473101/23854702/f7b36b76-07c8-11e7-9baf-fed444fbebfe.png)

Can you please help me out with this ?  

Thanks 
Gagan

**Comment by HalbergM** [comment]
I ran into the same issue and found that the webpack and webpack-dev-server dependencies were out of date.

Also, the loaders in the webpack.config.js now require the "-loader" suffix.

After I had updated those, I no longer receive any errors during development.
I have not tested it all the way through the build stage, however.

I made a pull request #6 for the changes

To fix the issues you are having:

**package.json** 
under `devDependencies`
change
```
"webpack": "2.1.0-beta.22",
"webpack-dev-server": "^2.1.0-beta.0"
```
to
```
"webpack": "^2.6.1",
"webpack-dev-server": "^2.4.5"
```

**webpack.config.js**
under `loaders:`
change
`loader: 'vue'`
and
`loader: 'babel'`
to
`loader: 'vue-loader'`
and
`loader: 'babel-loader'`
